src/Detector/PfmDetector.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Selective\ImageType\Detector;
+
+use Selective\ImageType\ImageType;
+use SplFileObject;
+
+/**
+ * Detector.
+ */
+final class PfmDetector implements DetectorInterface
+{
+    /**
+     * PFM identification.
+     *
+     * @param SplFileObject $file The image file
+     *
+     * @return ImageType|null The image type
+     */
+    public function detect(SplFileObject $file): ?ImageType
+    {
+        $file->rewind();
+        $bytes = strtoupper((string) $file->fread(2));
```

Comment:
The first 2 bytes will be `PF` or `Pf`.

To identify image format easily, we can let these 2 bytes string be upper string before checking magic bytes.

We can only check whether the bytes will be `PF`.